torch/_jit_internal.py

```diff
@@ -111,7 +111,7 @@ def weak_script_method(fn):
     return fn
 
 
-def boolean_dispatch(arg_name, arg_index, default, if_true, if_false):
+def boolean_dispatch(arg_name, arg_index, default, if_true, if_false, module_name=None, func_name=None):
```

Comment:
Anywhere this function is used should hook these names up, so these should just be required

Reply:
Ok I made the change